src/main/scala/circt/stage/ChiselStage.scala

```diff
@@ -13,6 +13,7 @@ import firrtl.{AnnotationSeq, EmittedVerilogCircuitAnnotation}
   *
   * @note The companion object, [[ChiselStage$]], has a cleaner API for compiling and returning a string.
   */
+@deprecated(pleaseSwitchToCIRCT, "Chisel 5")
```

Comment:
```suggestion
@deprecated(pleaseSwitchToCIRCT, "Chisel 3.6")
```
Deprecated version is "since"